MapboxCoreNavigation/Constants.swift

```diff
@@ -55,7 +50,7 @@ public let RouteControllerProgressDidChange = Notification.Name(MBRouteControlle
 /**
  Emitted when the alert level changes. This indicates the user should be notified about the upcoming maneuver.
  */
-public let RouteControllerAlertLevelDidChange = Notification.Name(MBRouteControllerAlertLevelDidChange)
+public let RouteControllerDidPassVoiceInstructionPoint = Notification.Name(MBRouteControllerDidPassVoiceInstructionPoint)
```

Comment:
Rename to `RouteControllerDidPassSpokenInstructionPoint` for consistency. Can you look through the PR for other places that should say “spoken” instead of “voice”?